demo-kotlin/app/src/main/java/com/example/placesdemo/PlaceDetailsAndPhotosActivity.kt

```diff
@@ -294,6 +295,13 @@ class PlaceAndPhotoTestActivity : AppCompatActivity() {
         findViewById<View>(R.id.loading).visibility = if (loading) View.VISIBLE else View.INVISIBLE
     }
 
+    private fun clearViews() {
+        responseView.setText(null)
```

Comment:
ah, yes (though I think you meant the reverse, that we should use the property access instead of the setter method). For the ImageViews, we'd still use the setter method right?

Reply:
Jep, I meant the other way around. The `ImageView` still uses the setter (the setter does a bit more than just setting the value, hence the reason.